pints/_mcmc/_population.py

```diff
@@ -327,10 +325,12 @@ def tell(self, fx):
                 # Update state
                 self._have_exchanged = True
 
+        # Return "accepted" any time the zero-chain moves
+        changed = accepted and self._i == 0
+        changed |= self._have_exchanged and (self._i == 0 or self._j == 0)
+
         # Return new point for chain 0
-        sample = np.array(self._current[0], copy=False)
-        sample.setflags(write=False)
-        return sample
+        return np.copy(self._current[0]), self._current_log_pdfs[0], changed
```

Comment:
Why do we return a copy of `self._current[0]` instead?

Reply:
self._current holds samples for several (tempered) chains, but you only want the samples from chain 0